app/src/main/res/layout-land/help_item.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data class="HelpItemBinding">
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.help.HelpViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
```

Comment:
Please see the mock there is a border of 1dp given to the list item. Seems like you may revert your last commit and add this border and this would fix over all.
<img width="1117" alt="small" src="https://user-images.githubusercontent.com/54615666/77282694-a5edd700-6cf0-11ea-8f77-3ecb609acf98.png">